pages/sign-in.tsx

```diff
@@ -3,27 +3,59 @@ import Input from "@/components/shared/forms/Input/Input";
 import Layout from "@/components/shared/layout/Layout";
 import Heading from "@/components/shared/typography/Heading/Heading";
 import Link from "next/dist/client/link";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as z from "zod";
+
+type FormData = z.infer<typeof schema>;
+
+const schema = z.object({
+  name: z.string().min(1, { message: "Введіть Ім'я" }),
+  email: z.string().min(1, { message: "Введіть email" }),
+  password: z
+    .string()
+    .min(8, { message: "Пароль має містити мінімум 8 символів" }),
+  confirmPassword: z.string(),
+});
```

Comment:
We have only 2 inputs for `email` and `password`
```suggestion
const schema = z.object({
  email: z.string().min(1, { message: "Введіть email" }),
  password: z
    .string()
    .min(8, { message: "Пароль має містити мінімум 8 символів" }),
});
```